dandi/consts.py

```diff
@@ -83,6 +83,12 @@
     "local-docker": dandi_instance(
         "http://localhost:8080", "http://localhost:8085", None
     ),
+    "local-docker-compose": dandi_instance(
```

Comment:
well, `local-docker` is also via compose.  This one is specifically for testing, so may be name it `"local-docker-tests"` or alike?

Reply:
Changed.